index.html

```diff
@@ -1,111 +1,579 @@
+        <button type="submit">whitelist now</button>
+      </section>
+      <section class="catalog">
+        <div>
+          <h2><span class="folder-icon"></span>Catalog</h2>
+          <div></div>
+          <a href="catalog.html" class="link">See all</a>
+          <button>&#8249;</button>
+          <button>&#8250;</button>
+        </div>
+        <div class="catalog-list">
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="catalog-lable">NEW</span>
+            <span class="catalog-info-price">$199</span>
+            <span class="catalog-name-product">Name Product</span>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="catalog-lable">NEW</span>
+            <span class="catalog-info-price">$199</span>
+            <span class="catalog-name-product">Name Product</span>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="catalog-lable">Sold out</span>
+            <span class="catalog-info-price">$199</span>
+            <span class="catalog-name-product">Name Product</span>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="catalog-lable">NEW</span>
+            <span class="catalog-info-price">$199</span>
+            <span class="catalog-name-product">Name Product</span>
+          </div>
+          <div class="catalog-info-item">
+            <img src="url" alt="text" />
+            <span class="catalog-lable">NEW</span>
+            <span class="catalog-info-price">$199</span>
+            <span class="catalog-name-product">Name Product</span>
+          </div>
+        </div>
+      </section>
+      <section class="blog">
+        <h2 class="blog-name">Our blog</h2>
+        <div class="blog-list">
+          <article class="blog-post">
+            <img src="#" alt="text" />
+            <span class="post-lable">NTF</span>
+            <div class="post-info">
+              <span class="post-author">Tony Nguyen</span>
+              <span class="post-date">20 Jun 2022</span>
             </div>
-
-            <div class="body">
-              <div class="profile-icon-box">
-                <p id="typeText"></p>
-                <div class="profile-icon">
-                    <img src="https://assets.codepen.io/4292193/ayakoyokoya_icon.png" /> <!-- Photo -->
-                </div>
-              </div>
-
-              <div class="name">
-                <p>AYAKO YOKOYA</p> <!-- Name -->
+            <div class="post-title">
+              <h3 class="post-title-name">Sonic live</h3>
+              <p class="post-content">
+                One September 22, 2021 A volutpat nulla eros, nulla ultrices nec
+                nulla imperdict
+              </p>
+              <a class="link" href="blog.html">Read more</a>
```

Comment:
This element doesn't belong to the 'post-content' block. Please move it outside the block